scanapp/tasks.py

```diff
@@ -24,21 +24,27 @@
 from __future__ import absolute_import, unicode_literals
 
 import json
+import logging
+import os
+import shutil
 import subprocess
-import requests
 from datetime import datetime
```

Comment:
These were actually automatically sorted by my PyCharm editor according to pep8 guidelines.